### PR TITLE
Anchoring a Cloth

### PR DESCRIPTION
**Pull request by BabylonJSGuide**
You can now anchor a cloth to a rigid body as in this demo https://adoring-kirch-5b50fe.netlify.com/anchor.html and if it is approved I am happy for it to be merged. 

However I have been trying for a more elegant solution to a problem, than the one I used, that came up when adding the anchors. I have raised this in [#5974](https://github.com/BabylonJS/Babylon.js/issues/5974)

### CODE REVIEW

**Comment by BabylonJSGuide** [comment]
Need help to understand the documentation errors, have looked at the methods mentioned and they have comments as far as I can see. What have I missed and what do I need to change?

File: src/Physics/Plugins/ammoJSPlugin.ts [hunk 1]
```diff
@@ -1145,6 +1150,24 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
         }
     }
 
+     /**
+     * Append an anchor to a soft object
+     * @param impostor, soft impostor to add anchor to
```
Comment (deltakosh) [inline src/Physics/Plugins/ammoJSPlugin.ts]: you should remove the comma I guess

Comment (BabylonJSGuide) [reply src/Physics/Plugins/ammoJSPlugin.ts]: Will do

Comment (BabylonJSGuide) [reply src/Physics/Plugins/ammoJSPlugin.ts]: Done

**Comment by BabylonJSGuide** [comment]
Have added a changed based on @TrevorDev PR, #5979

**Comment by BabylonJSGuide** [comment]
found small error will correct in a few hours

**Pull request merged**